landing-page/components/Footer.tsx

```diff
@@ -58,7 +58,7 @@ const linkGroups = [
       { name: 'LOI', link: 'https://cdn.helpwave.de/helpwave_letter_of_intent.pdf' },
       { name: 'Tech-Radar', link: '/tech-radar', openInCurrentTab: true },
     ],
-    Products: [
```

Comment:
Pretty sure this uppercase was on purpose, as the footer uses the key as input.

Reply:
Okay, i need to check again the Problem ist that i am using the Name AS translation key, so the orthography is importan